lib/shared/bucketing-assembly-script/assembly/eventQueue/eventQueue.ts

```diff
@@ -0,0 +1,197 @@
+import {
+    DVCEvent,
+    DVCPopulatedUser,
+    EventQueueOptions,
+    DVCRequestEvent,
+    UserEventsBatchRecord,
+    FlushPayload,
+    FeatureVariation
+} from '../types'
+import { jsonArrFromValueArray } from '../helpers/jsonHelpers'
+import { RequestPayloadManager } from './requestPayloadManager'
+
+export type AggEventQueue = Map<string, Map<string, Map<string, Map<string, i64>>>>
```

Comment:
it might help understanding to unpack this into multiple sub types that are named descriptively for each level of nesting